gui/controls/PopupButton.qml

```diff
@@ -0,0 +1,51 @@
+import QtQuick 2.11
+import QtQuick.Controls 2.4
+import ".."
+
+/*!
+    \qmltype PopupButton
+    \brief Simple styled button to be used in popups
```

Comment:
This is more specifically a popup radio button, right? Would it make sense to rename accordingly?

Reply:
Yes, I saw the radio button design only later. I am still a bit confused to where get the design specs from. I will update.